src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -4,50 +4,50 @@
   
     "power.toomanyorigins.overheat.name": "Aşırı Isınma",
-    "power.toomanyorigins.overheat.description": "Baktığın herhangi bir nesneden uzağa doğru roket zıplaması gerçekleştirebilirsin, bu süreç seni biraz incitir.",
-    "power.toomanyorigins.aerial_stability.name": "Hava Stabilliği",
+    "power.toomanyorigins.overheat.description": "Baktığın herhangi bir nesneden uzağa doğru roket zıplaması gerçekleştirebilirsin, bu süreç seni incitir.",
+    "power.toomanyorigins.aerial_stability.name": "Hava Dengesi",
     "power.toomanyorigins.aerial_stability.description": "Aşağıya doğru hızınız biraz azaltılmış.",
     "power.toomanyorigins.conductive.name": "İletken",
     "power.toomanyorigins.conductive.description": "Yıldırım tarafından çarpıldığında şarjlanırsın.",
     "power.toomanyorigins.stubby_legs.name": "Güdük Bacaklar",
-    "power.toomanyorigins.stubby_legs.description": "Kısa bacakların ayakta daha yavaş hareket etmeni sağlar.",
+    "power.toomanyorigins.stubby_legs.description": "Kısa bacakların yüzünden karada daha yavaş yürürsün.",
   
     "effect.toomanyorigins.charged": "Şarjlı",
     "death.attack.overheat": "%1$s içini fitilledi",
-    "death.attack.overheat.player": "%1$s, %2$s den kaçmaya çalışırken içini fitilledi.",
-    "death.attack.overheat.player.item": "%1$s, %3$s kullanan %2$s den kaçmaya çalışırken içini fitilledi.",
+    "death.attack.overheat.player": "%1$s, %2$s'den kaçmaya çalışırken içini fitilledi.",
+    "death.attack.overheat.player.item": "%1$s, %3$s kullanan %2$s'den kaçmaya çalışırken içini fitilledi.",
   
     "power.toomanyorigins.hover.name": "Süzül",
-    "power.toomanyorigins.hover.description": "Düşerken yükseklik kaybetmeden süzülebilirsiniz.",
+    "power.toomanyorigins.hover.description": "Düşeceğin zaman havada asılı kalabilirsin, bu şüreçte biraz acıkırsın.",
     "power.toomanyorigins.pollination.name": "Tozlaşma",
     "power.toomanyorigins.pollination.description": "Eğilmiyorken kemik tozu kullanmak yanlardaki 4 bloğa da etki eder.",
     "power.toomanyorigins.calming_aura.name": "Sakin Aura",
-    "power.toomanyorigins.calming_aura.description": "Senin tarafından çiftleştirilmiş hayvanlar bir daha çiftleşmek için beklemeye ihtiyaç duymaz.",
-    "power.toomanyorigins.stinging_pains.name": "Batırma Acısı",
-    "power.toomanyorigins.stinging_pains.description": "Düşman canlıları direkt olarak öldürdüğünde acıkırsın.",
+    "power.toomanyorigins.calming_aura.description": "Çiftleştirdiğin saldırgan olmayan hayvanların tekrar çiftleşebilmesi için beklemene gerek yok.",
+    "power.toomanyorigins.stinging_pains.name": "Batırmanın Acısı",
+    "power.toomanyorigins.stinging_pains.description": "Düşman canlıları doğrudan öldürdüğünde acıkırsın.",
     "power.toomanyorigins.unity.name": "Birlik",
-    "power.toomanyorigins.unity.description": "Açlık seviyene bağlı olarak maksimum can kaybedersin.",
+    "power.toomanyorigins.unity.description": "Maksimum canın, açlık seviyene bağlı.",
```

Comment:
was going to use "Azami" here instead of "Maksimum", but turns out Azami has arabic roots :/